### PR TITLE
Animations and Stuff

### PR DESCRIPTION
**Pull request by brandenbyers**
This should be everything; do we need anything else?

### CODE REVIEW

File: raffle.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import random
 import sys
 from random import shuffle
+totalWinners = 42
```
Comment (jetpacktuxedo) [inline raffle.py:5]: Y'all really killing me with this hardcoded global variable stuff.

This and the csv file should probably be handled by argparse.

Obviously we are on a bit of a time crunch, but we should clean this up for next time.

File: raffle.py [hunk 2]
```diff
@@ -10,22 +11,26 @@
...
+<!doctype html>
+<html lang=en>
+<head>
+<meta charset=utf-8>
+<title>SMKmeetup Giveaway Winners!</title>
+<link rel="stylesheet" href="raffle-style.css">
+</head>
+<body>
+<h2>SMKmeetup Giveaway Winners</h2>
+<ul class="multi-12">
```
Comment (jetpacktuxedo) [inline raffle.py:26]: We should probably use Jinja templates or something for this instead of building raw strings

Comment (brandenbyers) [reply raffle.py:26]: Jinja what? 😜

Comment (jetpacktuxedo) [reply raffle.py:26]: http://jinja.pocoo.org/

File: raffle.py (hunk 1 above)
Comment (brandenbyers) [reply raffle.py:5]: Totally my fault...I just wanted a real quick way to change the winner count day-of if anything changes. It is just a tiny script so I didn't see too much harm in it.

Comment (jetpacktuxedo) [reply raffle.py:5]: Nah, James did it with the argv stuff to begin with so there wasn't a great place for you to add it. It'll be fine for now, but we should refactor eventually.

**Pull request merged**